go/libkb/util.go

```diff
@@ -461,40 +461,42 @@ func RandHexString(prefix string, numbytes int) (string, error) {
 }
 
 func Trace(log logger.Logger, msg string, f func() error) func() {
-	// NOTE: Here and in all the similar functions below, we increment the log
-	// depth in the calls we make immediately, but *not* in the calls we defer
-	// to the caller. Those will (probably, hopefully) run in the caller's
```

Comment:
This note is wrong.

Reply:
Damn. I thought I tested this by hand. What actually happens?